pkg/test/verify/cluster.go

```diff
@@ -47,26 +47,36 @@ func AddToolchainClusterAsMember(t *testing.T, functionToVerify FunctionToVerify
 				require.NoError(t, err)
 				cachedToolchainCluster, ok := cluster.GetCachedToolchainCluster("east")
 				require.True(t, ok)
-				assert.Equal(t, cluster.Member, cachedToolchainCluster.Type)
-				if labels["namespace"] == "" {
-					assert.Equal(t, "toolchain-member-operator", cachedToolchainCluster.OperatorNamespace)
-				} else {
-					assert.Equal(t, labels["namespace"], cachedToolchainCluster.OperatorNamespace)
-				}
+
+				assert.Equal(t, labels["namespace"], cachedToolchainCluster.OperatorNamespace)
+
 				// check that toolchain cluster role label tenant was set only on member cluster type
 				require.NoError(t, cl.Get(context.TODO(), client.ObjectKeyFromObject(toolchainCluster), toolchainCluster))
-				expectedToolChainClusterRoleLabel := cluster.RoleLabel(cluster.Tenant)
-				_, found := toolchainCluster.Labels[expectedToolChainClusterRoleLabel]
```

Comment:
I'm still missing the verification of the role label

Reply:
adding it.